build.gradle

```diff
@@ -24,6 +24,8 @@ plugins {
     id 'com.google.cloud.tools.jib' version '2.2.0'
 }
 
+ext.dtestJar = System.getenv("DTEST_JAR") ?: "dtest-5.0.jar" // trunk is currently 5.0.jar - update when trunk moves
```

Comment:
Should we move this default to gradle.properties?

Reply:
I thought about that but it felt like, given this is the only place it's used, adding a level of indirection to change it seemed unnecessary.

Reply:
The default should be 4.0 jar

Reply:
The reason is sidecar supports 4.0 and above. The default jar should be 4.0, so that compiler enforces it by building with it. 
In order to track changes in 4.1 and trunk, there are integration test that runs with both configurations. 
There are 2 goals of those integration tests.
1. to catch any ABI-incompatible changes in Cassandra 4.1.
2. to discover any 5.0 specific changes required and add it to the corresponding sidecar "adapter".

Reply:
Also just note that the only reason this is included is to use for the dtest classes - we're not otherwise using the jar, so in theory we could just pick one, but I thought it would be good to use trunk so we would catch ABI changes in trunk more easily (and at one point 4.0 didn't even _work_ because it was missing some packages, but I've fixed that in the dtest jar build process now)